cddl/manifest.cddl

```diff
@@ -3,7 +3,7 @@
 ; For that reason it has been simplified for that use case:
 ;
 ; Removed unused extensions
-; Limited the number of possible integrated payloads to 5
+; Limited the number of possible integrated payloads to 6
```

Comment:
Please update the following line too:
 `; Limited the number of possible component identifiers to 5`

Reply:
Done